stimela/main.py

```diff
@@ -74,6 +74,7 @@ def resolve_command(self, ctx, args):
     help="Reset the configuration cache. First thing to try in case of strange configuration errors.",
 )
 @click.option("--simple", "-S", is_flag=True, help="Use simplified (single-line) displays.")
+@click.option("--no-progress", "-np", is_flag=True, help="Disable the progress display.")
```

Comment:
I feel these help strings could be a little clearer. These three options are in increasing order of boringness, correct? They can't be meaningfully combined? Let's try to clarify this in the CLI.

Reply:
I added `--no-progress` because `--boring` strips out colour. The main reason for having `--boring` was to make debugging interactively simple (if my memory serves), but the colours don't interfere with that. I will push an update in a moment which makes the options names more consistent i.e. `--simplify-display` and `--disable-display`. I think that keeping them as separate options with slightly more verbose help is better than trying to merge them into one. I am also wary of altering the behaviour of `--boring` as it seems surprisingly involved.